configuration/editors.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different Ways to Simplify Your Textual Configuration
+
+Currently there are several existing solutions, that can help you configuring your openHAB instance in a textual way.
+This documentation page can give you some guidance in choosing the right one for you and setting it up.
+
+{::options toc_levels="2"/}
```

Comment:
@Confectrician I didn't see that before. The default is `2..4` which would look better here, do you agree?

![grafik](https://user-images.githubusercontent.com/2870104/29773271-ec1e930e-8bfc-11e7-8de6-532e15fa53e4.png)

Reply:
Was a relict from the old "others" design. i have changed it and it comes with the link fix PR.